mediacapture-streams/enumerating-local-media-devices/mediadevices/api-present.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+<head>
+<title>getUserMedia: test that getUserMedia is present (with or without vendor prefix)</title>
+<link rel="author" title="Dr Alex Gouaillard" href="mailto:agouaillard@gmail.com"/>
+<link rel="help" href="https://www.w3.org/TR/mediacapture-streams/#mediadevices">
+<meta name='assert' content='Check that the mediaDevices.enumerateDevices() method is present.'/>
+<meta name='flags' content='dom'/>
+<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
```

Comment:
Won't this be included automatically? In general, this test is far more verbose than it needs to be, a lot of this is added automatically. In its current form, it could be reduced to
```HTML
<!doctype html>
<title>mediaDevices.enumerateDevices() returns a promise</title>
<script src=/resources/testharness.js></script>
<script src=/resources/testharnessreport.js></script>
<script>
assert_true(navigator.mediaDevices.enumerateDevices() instanceof Promise);
</script>
```

I'll defer to @hta, but at least I much prefer minimizing tests as much as possible so that the interesting bits stand out.